src/main/resources/static_config/static_action_groups.yml

```diff
@@ -87,6 +87,7 @@ read:
   allowed_actions:
   - "indices:data/read*"
   - "indices:admin/mappings/fields/get*"
+  - "indices:admin/resolve/index"
```

Comment:
any other action groups that should be granted `resolve` permission? For example `search`?

Reply:
Since `indices:admin/resolve/index` permission is an addition to the granularity of permission `indices:data/read/search`, granting it to all existing action groups originally owns `indices:data/read/search`

Reply:
I don't see any relationship here.